gui/functionality/manage_jobs.py

```diff
@@ -0,0 +1,94 @@
+from PyQt5.QtWidgets import QTableWidgetItem
+from idact import show_cluster, load_environment
+from idact.detail.slurm.run_scancel import run_scancel
+from idact.detail.slurm.run_squeue import run_squeue
+
+from gui.functionality.idact_app import IdactApp, WindowType
+from gui.helpers.decorators import addToClass
+from gui.helpers.worker import Worker
+
+
+class ManageJobs:
```

Comment:
I'm not sure about naming of this class. I think it should be more like `JobManager` or something similar.

Reply:
Other classes are named similarly - `AddCluster`, `RemoveCluster`. So as it should be consistent we should rename all classes or left it as it is.

Reply:
Oh, good point. So I think it should stay as it is.

Reply:
Oh, good point. So I think it should stay as it is.